crates/generate-json-rpc-spec/Cargo.toml

```diff
@@ -16,6 +16,8 @@ tempfile = "3.3.0"
 tokio = { version = "1.18.2", features = ["full"] }
 
 sui = { path = "../sui" }
+sui-rpc = { path = "../sui-rpc" }
+sui-rpc-api = { path = "../sui-rpc-api" }
```

Comment:
Lets have these have more descriptive names. specifically since this is talking about json-rpc these should probably be named `sui-json-rpc`